CHANGES/7572.feature

```diff
@@ -0,0 +1 @@
+Expose MANIFEST.json and DIGEST.json at CollectionVersion endpoint
```

Comment:
This and title should be updated from DIGEST.json to FILES.json, I'm not aware of any DIGEST.json